javaslang/src-gen/main/java/javaslang/Tuple0.java

```diff
@@ -127,12 +127,12 @@ private Object readResolve() {
     }
 
     @Override
-    public <T> Tuple1<T> prepend(T value) {
+    public <T> Tuple1<T> append(T value) {
```

Comment:
Preferences on the semantic ordering of methods may vary - depending of the person who is looking at the code.

I try to keep alphabetic ordering (which is, however, not continuously applied yet).

A (more or less) precise specification on which order we prefer you find in the file [CONTRIBUTION.md (section 'File structure')](https://github.com/javaslang/javaslang/blob/master/CONTRIBUTING.md#file-structure).

---

Side-note: Regarding  `append` and `prepend` I have come to a conclusion: we will remove them again. The builder should not be baked into the tuple itself. Also we block the evolution of Tuple (e.g. increasing the max number of tuples) by letting Tuple8.append/prepend throw.

@ashrko619 please don't feal 😢 about it (removing append/prepend again). It is hard to make correct API decisions. I will try to do deeper investigations before you create a PR next time!